packages/generator/src/v1.1/generatePerQueryDataFiles.ts

```diff
@@ -26,24 +34,119 @@ export async function generatePerQueryDataFiles(
   fs: MinimalFs,
   outDir: string,
   importExt: string = "",
+  v2: boolean = false,
 ) {
   await fs.mkdir(outDir, { recursive: true });
   await Promise.all(
     Object.values(ontology.queryTypes).map(async query => {
       const objectTypes = getObjectTypesFromQuery(query);
-      await fs.writeFile(
-        path.join(outDir, `${query.apiName}.ts`),
-        await formatTs(`
-        import { QueryDefinition } from "@osdk/api";
-
-        export const ${query.apiName} = ${
-          JSON.stringify(wireQueryTypeV2ToSdkQueryDefinition(query))
-        } satisfies QueryDefinition<"${query.apiName}", ${
-          objectTypes.length > 0
-            ? objectTypes.map(apiName => `"${apiName}"`).join("|")
-            : "never"
-        }>;`),
-      );
+      const importObjects = objectTypes.length > 0
```

Comment:
I need to refactor this a bunch (especially the duplicated parts), just did it all here to make debugging the code gen easier

Reply:
Cool